commands/bmc.nicole

```diff
@@ -152,14 +152,45 @@ function cmd_datetime_method {
 # Set system time manually
 #   YYYY-MM-DD-HH-mm-SS - Date and time to set
 function cmd_datetime_set {
+  local datestr="${1-}"
```

Comment:
Likely you meant `${1:-}`

Reply:
They are the same, see https://infoheap.com/bash-set-default-value-if-a-variable-is-empty/